### PR TITLE
Added Django 1.7 and up above documentation usage

### PR DESCRIPTION
**Pull request by RaidoS**
more about this features https://docs.djangoproject.com/en/1.9/topics/migrations/#data-migrations

if some thing wrong commited, say plz, i`ll change it.

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ in Django is declared through the model's Meta class. An example ``models.py``:
 	class SortableBook(models.Model):
 	    title = models.CharField('Title', null=True, blank=True, max_length=255)
 	    my_order = models.PositiveIntegerField(default=0, blank=False, null=False)
-
```
Comment (jrief) [inline docs/source/usage.rst]: why did you remove these tabs, they help me to see which blocks belong together

**Comment by jrief** [comment]
I think we can drop support for south altogether. After all Django-1.6 is deprecated anyway.

File: docs/source/usage.rst (hunk 1 above)
Comment (RaidoS) [inline docs/source/usage.rst]: sorry, i was hurry. because doing work with my own project . And was afraid to forget
about pull request.

**Comment by RaidoS** [comment]
Should me re pull request? With deprecation notes? (and with your code styling)

**Comment by jrief** [comment]
@RaidoS yes, please

**Comment by RaidoS** [comment]
i`ll do send new request. I want re fork it, because something wrong with my local git history.
